daemon/api.go

```diff
@@ -416,7 +418,9 @@ func loginUser(c *Command, r *http.Request, user *auth.UserState) Response {
 		}, nil)
 	}
 
-	macaroon, discharge, err := store.LoginUser(loginData.Email, loginData.Password, loginData.Otp)
```

Comment:
The alternative to this would be to make "LoginUser()" part of StoreService - which probably makes sense?

Reply:
Thanks, I will probably do this as a followup, it is quite a bit of extra churn :/